CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - **Breaking**: In formula-android, replacing lastEntry() with visibleState() in FragmentFlowState.
 - Allow down-casting `ActivityStoreContext<MyActivity>` to `ActivityStoreContext<FragmentActivity>`
 - Enable global fragment error logging.
+- **Breaking**: Replacing `Renderer` with a `Render` class. Also, breaking changes in `RenderView`.
```

Comment:
Are we going to make a new release with this change?

Reply:
Good idea, it makes sense to do a `0.6.0` with all these changes.